node-check/osgvo-additional-htcondor-config

```diff
@@ -59,7 +59,8 @@ add_config_line HasExcessiveLoad "LoadAvg > 2*DetectedCpus + 2"
 add_condor_vars_line HasExcessiveLoad "C" "-" "+" "N" "Y" "-"
 
 # out of disk space (https://opensciencegrid.atlassian.net/browse/OSPOOL-4)
-add_config_line RESERVED_DISK "3000000"
+# note: Unlike DISK, RESERVED_DISK is in megabytes
```

Comment:
```suggestion
# note: Unlike DISK, RESERVED_DISK is in megabytes.
# As of July 2022, the HTCondor manual incorrectly states this is in kilobytes.
# This change was verified by examining the code.
```

Reply:
Dang, you're right. I'll make a JIRA ticket.